### PR TITLE
New package COAA.PlanePlotter

### PR DESCRIPTION
**Pull request by blackzafiqz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.
request from #115736
-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/116546&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
Hi @blackzafiqz,

Is there an installer switch to bypass the above message? Alternatively, should NTP be a dependency?

**Comment by AsciiWolf** [comment]
@stephengillie @blackzafiqz Any chance this PR could be re-opened and the NTP issue (that will most likely not happen on 99% of actual end-user computers) waived?

**Comment by stephengillie** [comment]
Hi @AsciiWolf,

> @stephengillie @blackzafiqz Any chance this PR could be re-opened and the NTP issue (that will most likely not happen on 99% of actual end-user computers) waived?

Is the assumption about 99% of end-user computers based on an assumption that only current users will install the software through the package manager, and thus will already have the dependency preinstalled? Please understand that we may have users whose hobby is downloading and installing random software, and if the dependency is missing from their system, then these new users of this software might have a poor experience. Much of the work we do here is to ensure the best experience that we can, for installing and upgrading software.  

NTP isn't currently among our manifests, so one will need to be added to the repo before it can be used as a dependency.

**Comment by AsciiWolf** [comment]
> NTP isn't currently among our manifests, so one will need to be added to the repo before it can be used as a dependency.

If I remember it correctly, this issue is probably related to a network limitation in the CI vm, not a dependency issue. NTP is a part of Windows since Windows 9x as the "Windows Time service".

**Comment by AsciiWolf** [comment]
Ah, you are right, it looks like PlanePlotter for some reason needs [this component](https://www.meinbergglobal.com/english/sw/ntp.htm).

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 876.)

**Comment by AsciiWolf** [comment]
@stephengillie https://github.com/microsoft/winget-pkgs/issues/148222

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @blackzafiqz,

`MeinbergGlobal.NTP` was just added in #148225. I tried using it as a dependency for this package, but my VM had this same error (below) on both tries. Maybe the validation pipeline will have better luck? 

> 2024-04-08 20:56:52.845 [REPO] Opening database for ReadWrite at 'C:\Users\User\AppData\Local\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\StoreEdgeFD\installed.db'
2024-04-08 20:56:52.846 [REPO] Opened SQLite Index with version [1.7], last write [2024-03-15 12:43:21.000]
2024-04-08 20:56:52.847 [REPO] Sending http GET request to: https://storeedgefd.dsx.mp.microsoft.com/v9.0/packageManifests/MeinbergGlobal.NTP?Market=US
2024-04-08 20:56:52.938 [REPO] Response status: 200
2024-04-08 20:56:52.949 [REPO] Opening database for ReadWrite at 'C:\Users\User\AppData\Local\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\Microsoft.Winget.Source_8wekyb3d8bbwe\installed.db'
2024-04-08 20:56:52.950 [REPO] Opened SQLite Index with version [1.7], last write [2024-04-08 20:54:51.000]
2024-04-08 20:56:52.957 [CLI ] Terminating context: 0x8a150104 at C:\__w\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DependenciesFlow.cpp:129

File: manifests/c/COAA/PlanePlotter/6.6.5.3/COAA.PlanePlotter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: COAA.PlanePlotter
+PackageVersion: 6.6.5.3
+Installers:
```
Comment (stephengillie) [inline manifests/c/COAA/PlanePlotter/6.6.5.3/COAA.PlanePlotter.installer.yaml:6]: I meant for this to be part of the previous review, but apparently forgot to hit "Start a review". Apologies.  

```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: MeinbergGlobal.NTP
Installers:
```

**Comment by blackzafiqz** [comment]
I will add to the in 4 hours. Don't have my pc right now

File: manifests/c/COAA/PlanePlotter/6.6.5.3/COAA.PlanePlotter.installer.yaml (hunk 1 above)
Comment (AsciiWolf) [reply manifests/c/COAA/PlanePlotter/6.6.5.3/COAA.PlanePlotter.installer.yaml:6]: This thread was marked as resolved, however it still needs to be added.

@blackzafiqz

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 876.)

**Comment by AsciiWolf** [comment]
Still not fixed. Please see my comment [here](https://github.com/microsoft/winget-pkgs/pull/116546#discussion_r1557541841).

**Comment by blackzafiqz** [comment]
sorry i didnt git add xD

**Comment by blackzafiqz** [comment]
I will try to fix on next week

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-04-10 22:20:06.797 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!6891535B: (caller: 68811A80) LogHr(1) tid(1c80) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!6895B5DD: (caller: 689F12BF) Exception(1) tid(1c80) 8A15005E ] 
> 2024-04-10 22:20:06.797 [REPO] Failed to open available source: msstore
> 2024-04-10 22:20:14.136 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!52BC535B: (caller: 52AC1A80) LogHr(1) tid(c90) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!52C0B5DD: (caller: 52CA12BF) Exception(1) tid(c90) 8A15005E ] 
> 2024-04-10 22:20:14.136 [REPO] Failed to open available source: msstore

(Automated response - build 876.)

**Comment by blackzafiqz** [comment]
i just force push again?

**Pull request closed without merging**